examples/jest_integration/integration.fuzz.js

```diff
@@ -14,17 +14,51 @@
  * limitations under the License.
  */
 
-/* eslint no-undef: 0 */
+/* eslint no-undef: 0, no-constant-condition: 0, @typescript-eslint/no-var-requires:0 */
 
-// eslint-disable-next-line @typescript-eslint/no-var-requires
 const target = require("./target");
 
 describe("My describe", () => {
-	it.fuzz("My regression test", async (data) => {
+	it.fuzz("My fuzz test", (data) => {
 		target.fuzzMe(data);
 	});
 
-	it.fuzz("My other regression test", (data) => {
-		target.fuzzMe(data);
+	it.fuzz("My callback fuzz test", (data, done) => {
+		target.callbackFuzzMe(data, done);
+	});
+
+	it.fuzz("My async fuzz test", async (data) => {
+		await target.asyncFuzzMe(data);
+	});
+
+	// In regression mode sync timeouts can not be detected, as the main event
+	// loop is blocked and registered timeout handlers can not fire.
+	// This is not only the case in regression test mode, but also during
+	// fuzzing runs. As the main event loop is blocked, no errors can be
+	// propagated to Jest. But the timeout set in libFuzzer will trigger a
+	// finding and shut down the whole process with exit code 70.
+	it.skip.fuzz("Sync timeout", () => {
```

Comment:
Probably there is not much we can do here, at least not in a clean way. 

I added checks for multiple calls to the `done` callback which resolve the initial promise, so that Jest could finish the tests. In the synchronous case the event loop is still blocked by the endless loop and so the resolved promise can not be processed, hence, everything is stuck. In the async case we can not notify the fuzzer thread about the second invocation, so it keeps on running until the configured timeout. I also added a few comments for the new checks, clarifying why the test can not be shut down cleanly. At least no OOM is generated anymore. 

I think this is quite a corner case and the environment does not support a clean solution. I would like to try it out this way and see if problems arise.